src/custom/state/orders/updaters/UnfillableOrdersUpdater.ts

```diff
@@ -40,8 +40,8 @@ async function _getOrderPrice(chainId: ChainId, order: Order, strategy: GpPriceS
     quoteToken = order.sellToken
   }
 
-  const isNativeSellSwap = order.sellToken === NATIVE_CURRENCY_BUY_ADDRESS
-  if (isNativeSellSwap) {
+  const isEthFlow = order.sellToken === NATIVE_CURRENCY_BUY_ADDRESS
```

Comment:
we can't pass ethflow from state here as this checks orders in state that are not related to the current pair